### PR TITLE
Solved issue #14: The plugin now sends metrics to Kafka in JSON type,…

### PR DESCRIPTION
**Pull request by cr2025x1**
… instead of raw byte[] from processor.

I took some codes from snap-plugin-publisher-file to convert metrics into JSON type.

https://github.com/intelsdi-x/snap-plugin-publisher-kafka/issues/14

### CODE REVIEW

**Comment by IzabellaRaulin** [comment]
Hello @cr2025x1 , thank you for your contribution. Could You format the file /kafka/kafka.go? - that the reason why the Travis checks have been failed. You can do that by executing this command in plugin dir:  
`go fmt kafka/kafka.go`

Please amend your pull request after this format update. 

Besides that, it LGTM.

**Comment by cr2025x1** [comment]
I formatted /kafka/kafka.go with "go fmt kafka/kafka.go" command as requested.

File: kafka/kafka.go [hunk 1]
```diff
@@ -47,10 +53,38 @@ func NewKafkaPublisher() *kafkaPublisher {
...
+		return fmt.Errorf("Error while marshalling metrics to JSON: %v", err)
+	}
+
+	// Inserted codes end
+
 	topic := config["topic"].(ctypes.ConfigValueStr).Value
 	brokers := parseBrokerString(config["brokers"].(ctypes.ConfigValueStr).Value)
 	//
-	err := k.publish(topic, brokers, content)
+	//	err := k.publish(topic, brokers, content)
```
Comment (andrzej-k) [inline kafka/kafka.go]: I'd suggest removing this line (code which is commented out).

**Comment by andrzej-k** [comment]
Nice work @cr2025x1 - only one comment from my side (kafka.go:86)

**Comment by IzabellaRaulin** [comment]
LGTM

**Pull request merged**